src/database/migrations/20240514075609-add-default-entities.js

```diff
@@ -7,119 +7,120 @@ module.exports = {
-				},
-				{
-					value: 'cc_by_sa',
-					label: 'CC BY SA',
-				},
-				{
-					value: 'cc_by_nc_sa',
-					label: 'CC BY NC SA',
-				},
-			],
-		}
+		const entitiesArray = [
+			{
+				entityType: 'resources',
+				entities: [
+					{ value: 'projects', label: 'Project' },
+					{ value: 'observations', label: 'Observation' },
+					{ value: 'observation_with_rubrics', label: 'Observation with rubric' },
+					{ value: 'surveys', label: 'Survey' },
+					{ value: 'programs', label: 'Program' },
+				],
+				has_entities: true,
+				validation: { required: true },
+				model: 'resources',
+			},
+			{
+				entityType: 'categories',
+				entities: [
+					{ value: 'teachers', label: 'Teachers' },
+					{ value: 'students', label: 'Students' },
+					{ value: 'community', label: 'Community' },
+					{ value: 'schoolProcess', label: 'School Process' },
+					{ value: 'infrastructure', label: 'Infrastructure' },
+					{ value: 'educationLeader', label: 'Education Leader' },
+					{ value: 'other', label: 'Other' },
+				],
+				has_entities: true,
+				validation: { required: true },
+				model: 'projects',
+			},
+			{
+				entityType: 'recommended_for',
+				entities: [
+					{ value: 'hm', label: 'HM' },
+					{ value: 'ht', label: 'HT' },
+					{ value: 'teachers', label: 'Teachers' },
+					{ value: 'educationLeader', label: 'Education Leader' },
+				],
+				has_entities: true,
+				validation: { required: true },
+				model: 'projects',
+			},
+			{
+				entityType: 'languages',
+				entities: [
+					{ value: 'en', label: 'English' },
+					{ value: 'hi', label: 'Hindi' },
+				],
+				has_entities: true,
+				validation: { required: true },
+				model: 'projects',
+			},
+			{
+				entityType: 'licenses',
+				entities: [
+					{ value: 'cc_by_4.0', label: 'CC BY 4.0' },
+					{ value: 'cc_by_nc', label: 'CC BY NC' },
+					{ value: 'cc_by_nc_nd', label: 'CC BY NC ND' },
+					{ value: 'cc_by_nd', label: 'CC BY ND' },
+					{ value: 'cc_by_sa', label: 'CC BY SA' },
+					{ value: 'cc_by_nc_sa', label: 'CC BY NC SA' },
+				],
+				has_entities: true,
+				validation: { required: true },
+				model: 'projects',
+			},
+			{
+				entityType: 'title',
+				entities: '',
+				has_entities: false,
+				validation: { regex: `[^A-Za-z0-9 <>_&-]`, required: true },
+				model: 'projects',
+			},
+			{
+				entityType: 'objective',
+				entities: '',
+				has_entities: false,
+				validation: { regex: `[^A-Za-z0-9 <>_&-]`, required: true },
+				model: 'projects',
+			},
+			{
+				entityType: 'keywords',
+				entities: '',
+				has_entities: false,
+				validation: { regex: `[^A-Za-z0-9 <>_&-]`, required: true },
+				model: 'projects',
+			},
+			{
+				entityType: 'tasks',
+				entities: '',
+				has_entities: false,
+				validation: { required: true },
+				model: 'projects',
+			},
+			{
+				entityType: 'name',
+				entities: '',
+				has_entities: false,
+				validation: { regex: `[^A-Za-z0-9 <>_&-]`, required: true },
```

Comment:
@ankit1shahu  Can we recheck this regular exp? 
this regex will match any character that is not a letter (upper or lowercase), digit, space, <, >, _, &, or -.